automation_tools/__init__.py

```diff
@@ -1015,12 +1015,9 @@ def setup_foreman_discovery(sat_version):
         return
 
     if sat_version == 'upstream-nightly':
-        run('foreman-installer --enable-foreman-plugin-discovery '
-            '--enable-foreman-proxy-plugin-discovery')
-        run('yum install -y {0}'.format(' '.join(packages)), warn_only=True)
```

Comment:
The rpms are pulled in by installer itself. No need to preinstall them. 
No sense to install them after installer run since they are installed already.

Except  CLI ```tfm-rubygem-hammer_cli_foreman_discovery```
This rpm is now installed by ```upstream_install()``` - see ```hammer_plugins``` list

Reply:
it is not there - fixed by PR #669